packages/experimental/Button/SPEC.md

```diff
@@ -4,6 +4,10 @@
 
 The `Button` component enables users to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.
 
+## Requirements
+
+If using FURN's theming, the `Button` requires use of the `ThemeProvider` from `@fluentui-react-native/theme` to work properly with themes. Please see [this page](https://github.com/microsoft/fluentui-react-native/blob/master/docs/pages/Guides/UpdateThemeProvider.md) for information on updating your `ThemeProvider` if using the version from `@uifabricshared/theming-react-native`.
```

Comment:
I figure since we'll probably keep the template the same between components, it wouldn't be too bad to remember to put this on top of every control. Since our controls will be a mixed bag until we get every control ported, I thought that having the prerequisite close to each control rather than in our docs folder somewhere would make it more visible.